test/Miscellaneous/rlexe.xml

```diff
@@ -1,79 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <regress-exe>
-<!-- ByteCode Header verification tests begin -->
-  <test>
-    <default>
-      <files>../../lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js</files>
-      <baseline>../../lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js.bc.32b.h</baseline>
-      <compile-flags>-GenerateLibraryByteCodeHeader -JsBuiltIn -LdChakraLib</compile-flags>
```

Comment:
These tests to check if the bytecode is up to date no longer work - I need to write a new test.